publishing/datasetAPI/put_instance_test.go

```diff
@@ -139,13 +145,16 @@ func TestSuccessfullyPutInstance(t *testing.T) {
 					checkEditionDoc(datasetID, instanceID, edition.Next)
 
 					Convey("and the dataset_id, edition and version values are set a properties on the neo4j instance node", func() {
-						neoDatasetID, neoEdition, neoVersion, err := neo4j.GetInstanceProperties(cfg.Neo4jAddr, instanceID)
+
+						instanceProps, err := neo4JStore.GetInstanceProperties(instanceID)
 						if err != nil {
-							t.Error(err)
+							log.ErrorC("failed to get properties from neo4j instance node", err, nil)
```

Comment:
Think about using `t.Errorf(...)` like on line 112 and 175

Reply:
Done